dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java

```diff
@@ -34,18 +36,29 @@ public class DumpMetadataTask extends AbstractTask<Void>
   @Nullable private final ConnectorArguments arguments;
   private final String format;
 
+  @Nullable private final ImmutableList<String> connectors;
+
   public DumpMetadataTask(@Nonnull ConnectorArguments arguments, @Nonnull String format) {
-    this(format, arguments);
+    this(format, arguments, /* connectors= */ null);
+  }
+
+  public DumpMetadataTask(
```

Comment:
In the long run, it is worthwhile to unify the argument order. For now I think @misolt 's suggestion is the best way to go.

Reply:
I've replaced one constructor with a factory method, because replacing the most popular one would require refactoring in all the call-sites, which is unrelated to the change in this PR and can be done in a separate PR.

The order of the parameters is now the same.